Library/Formula/libfreenect2.rb

```diff
@@ -0,0 +1,128 @@
+class Libfreenect2 < Formula
+  desc "Drivers/Example for Kinect V2"
+  homepage "https://github.com/OpenKinect/libfreenect2"
+  url "https://github.com/ybott/libfreenect2/archive/v0.0.1-alpha.tar.gz"
+  sha256 "704572b2848c38499ef934d9445d275033eb94d245aa60706eea2179ab2629c2"
+
+  depends_on "libusb"
+  depends_on "nasm" => :optional
+  depends_on "jpeg-turbo" => :optional
+  depends_on "homebrew/science/opencv" => :optional
+  depends_on "homebrew/versions/glfw3" => :optional
+  depends_on "pkg-config" => :build
+  depends_on "cmake" => :build
+
+  patch :DATA
```

Comment:
What is this patch for? Did you submit it upstream? Could you add a comment above this line explaining why it’s necessary? Thanks!